cmd/config-current.go

```diff
@@ -663,6 +663,24 @@ func applyDynamicConfig(ctx context.Context, objAPI ObjectLayer, s config.Config
 		return fmt.Errorf("Unable to apply scanner config: %w", err)
 	}
 
+	// Logger webhook
```

Comment:
You can remove this from `lookupConfigs`. `applyDynamicConfig` is called at the end.

Reply:
Done.